packages/asset/contracts/AssetCreate.sol

```diff
@@ -301,4 +301,6 @@ contract AssetCreate is
     {
         return ERC2771HandlerUpgradeable._msgData();
     }
+
+    uint256[47] private __gap;
```

Comment:
Does not include reserved slots for mappings?

Reply:
Same as above, it was my misconception.